### PR TITLE
#2838 - Bonds between atoms are not centered and not drawn symmetrically

### PR DESCRIPTION
**Pull request by AKZhuk**
## Check list
- [x] unit-tests written
- [x] e2e-tests written
- [ ] documentation updated
- [x] PR name follows the pattern `#1234 – issue name`
- [x] branch name doesn't contain '#'
- [x] PR is linked with the issue
- [x] base branch (master or release/xx) is correct
- [x] task status changed to "Code review"

All failed tests in the pipeline are successfully passed locally
[AutoTests.docx](https://github.com/epam/ketcher/files/12366910/AutoTests.docx)
[report.zip](https://github.com/epam/ketcher/files/12367077/report.zip)

### CODE REVIEW

**Review by MartaWilliams — APPROVED** [review]

File: packages/ketcher-core/src/application/render/restruct/rebond.ts [hunk 1]
```diff
@@ -651,24 +657,35 @@ function getBondPath(
...
+        newHalfBonds[1],
         bond,
         shiftA,
         shiftB,
         isSnapping,
       );
       break;
     case Bond.PATTERN.TYPE.ANY:
-      path = draw.bondAny(render.paper, hb1, hb2, render.options, isSnapping);
+      console.log(hb1.p, hb2.p);
```
Comment (StarlaStarla) [inline packages/ketcher-core/src/application/render/restruct/rebond.ts:672]: I am not sure should we keep console.log  here?

File: packages/ketcher-core/src/application/render/restruct/rebond.ts [hunk 2]
```diff
@@ -651,24 +657,35 @@ function getBondPath(
...
+      path = draw.bondAny(
+        render.paper,
+        newHalfBonds[0],
+        newHalfBonds[1],
+        render.options,
+        isSnapping,
+      );
       break;
     case Bond.PATTERN.TYPE.HYDROGEN:
+      newHalfBonds = util.updateHalfBondCoordinates(hb1, hb2, 1);
```
Comment (StarlaStarla) [inline packages/ketcher-core/src/application/render/restruct/rebond.ts:684]: maybe we could create meaningful constants for 1, -1 instead of numbers here

Comment (AKZhuk) [reply packages/ketcher-core/src/application/render/restruct/rebond.ts:684]: We have a rule https://eslint.org/docs/latest/rules/no-case-declarations 
that does not allow declare variables inside the case statement

Comment (StarlaStarla) [reply packages/ketcher-core/src/application/render/restruct/rebond.ts:684]: yes, but could we declare variables elsewhere

File: packages/ketcher-core/src/application/render/restruct/rebond.ts (hunk 1 above)
Comment (AKZhuk) [reply packages/ketcher-core/src/application/render/restruct/rebond.ts]: removed

File: packages/ketcher-core/src/application/render/restruct/rebond.ts (hunk 2 above)
Comment (AKZhuk) [reply packages/ketcher-core/src/application/render/restruct/rebond.ts]: done

**Review by StarlaStarla — APPROVED** [review]
Great job!

**Comment by ViktoriaTkacheva** [comment]
In two bonds (Aromatic Bond, Single/Aromatic Bond) the pattern is not symmetrical - the right dash is shorter than the left

image

**Pull request merged**